src/Impression.py

```diff
@@ -14,8 +14,8 @@ class ImpressionOpportunity:
     estimated_CTR: np.float32
     price: np.float32
     second_price: np.float32
-    outcome: np.bool
-    won: np.bool
+    outcome: bool
```

Comment:
Is the existing np.bool creating any issues for you?

Reply:
Numpy throws the warning: https://stackoverflow.com/questions/67779374/deprecationwarning-np-bool